spk/tcl/src/installer.sh

```diff
@@ -6,6 +6,9 @@ DNAME="Tcl"
 
 # Others
 INSTALL_DIR="/usr/local/${PACKAGE}"
+PACKAGE_DIR="/var/packages/${PACKAGE}/target"
```

Comment:
Previous comment still applies. 
Logfiles are allowed of course, but please put in in the package dir: `$(INSTALL_DIR)/var`. Although logging when a symlink is created has quite limited added value imo.